### PR TITLE
Fix 1165 unix test 0.27

### PR DESCRIPTION
**Pull request by clanmills**
See #1165.  All the issues are caused by binary output and have been fixed.  The purpose of the exiv2 command-line option --binrary has been clarified in man/man1/exiv2.

### CODE REVIEW

File: man/man1/exiv2.1 [hunk 1]
```diff
@@ -211,7 +211,7 @@ or 'm'(ute). The default log-level is 'w'. \fB\-Qm\fP is equivalent
 to \fB\-q\fP. All log messages are written to standard error.
 .TP
 .B \-b
-Show large binary values (default is to suppress them).
+The output from large binary values is to restricted by default.  When outputting the value of a large value such as MakerNote, the default behaviour is to restrict the data and end with "...".  The option --binary enables you to see all the data.
```
Comment (piponazo) [inline man/man1/exiv2.1:214]: Thanks for the explanation!

**Review by piponazo — DISMISSED** [review]
Great work Robin. In general it looks and I'm approving the changes. Nonetheless I made some comments and questions.

File: man/man1/exiv2.1 (hunk 1 above)
Comment (clanmills) [reply man/man1/exiv2.1:214]: I never knew what that did either.  It's now useful.

If we every really want to provide an option to output aw/binary data, we can use the option --raw/-R.  It's a pity that Andreas called that --binary/-b, you should think of it as "bloody long output!".

File: src/value.cpp [hunk 2]
```diff
@@ -1191,7 +1205,7 @@ namespace Exiv2 {
                    time_.hour, time_.minute, time_.second,
                    plusMinus, abs(time_.tzHour), abs(time_.tzMinute));
 
-        enforce(wrote == 11, Exiv2::kerUnsupportedTimeFormat);
+        // enforce(wrote == 11, Exiv2::kerUnsupportedTimeFormat);
```
[the remark this answers is not in the document]
Comment (clanmills) [reply src/value.cpp:1209]: Good Question.  I can't remember.  I think it was failing to compile somewhere, so I commented it off with the intention to investigate.  And guess what?  I forgot.  I'll investigate.

[the remark this answers is not in the document]
Comment (clanmills) [reply src/value.cpp]: I've restored the enforce and that builds OK (on Linux/macOS/sunOS/FreeBSD/NetBSD).  Don't know why I commented it off.  I'll build/test on MinGW/Cygwin and msvc2019 before merging.

**Pull request merged**